config/application.yml.default

```diff
@@ -22,6 +22,7 @@ aamva_supported_jurisdictions: '["AL","AR","AZ","CO","CT","DC","DE","FL","GA","H
 aamva_verification_request_timeout: 5.0
 aamva_verification_url: https://example.org:12345/verification/url
 all_redirect_uris_cache_duration_minutes: 2
+allowed_ialmax_providers: '["https://eauth.va.gov/isam/sps/saml20sp/saml20","https://myttbaccount.ttb.gov/realms/myTTB"]'
```

Comment:
I would consider leaving the prod value blank and putting the "real" values in S3 only

Reply:
oh i am happy to do that but don't know how -- will you show me? (we can do it on wednesday, i'm not merging this in today)

Reply:
values were added 07/06